packages/ember-runtime/lib/mixins/array.js

```diff
@@ -214,6 +216,14 @@ export default Mixin.create(Enumerable, {
 
   // optimized version from Enumerable
   contains(obj) {
+    if (isEnabled('ember-runtime-enumerable-includes')) {
+      deprecate(
+        '`Enumerable#contains` is deprecated, use `Enumerable#includes` instead.',
+        false,
+        { id: 'ember-runtime.enumerable-contains', until: '3.0.0', url: 'http://emberjs.com/deprecations/v2.x#toc_enumerable-contains' }
```

Comment:
I did not change deprecation id because feature is about `includes` but deprecation is about  `contains`.

But I don't know if it is accepted. Let me know